paths/reports/locales/index.yaml

```diff
@@ -0,0 +1,54 @@
+---
+summary: List Locale Reports
+description: List all locale reports for the given project
+operationId: reports/locales/list
+tags:
+  - Reports
+parameters:
+  - "$ref": "../../../parameters.yaml#/X-PhraseApp-OTP"
+  - "$ref": "../../../parameters.yaml#/project_id"
+  - "$ref": "../../../parameters.yaml#/page"
+  - "$ref": "../../../parameters.yaml#/per_page"
+  - "$ref": "../../../parameters.yaml#/locale_code"
```

Comment:
is locale optional? does the server return a list of reports for all the locales then?

Reply:
Yes, paginated. And the locale code is an option to Filter locales by their Code across projects